deployment/main.tf

```diff
@@ -0,0 +1,116 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "aws_ecs_cluster" "this" {
+  name = "akka-app-cluster"
+  tags = {
+    Project = "aipcc-akka"
+  }
+}
+
+resource "aws_iam_role" "ecs_task_execution" {
+  name = "ecsTaskExecutionRole"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [{
+      Effect = "Allow",
+      Principal = {
+        Service = "ecs-tasks.amazonaws.com"
+      },
+      Action = "sts:AssumeRole"
+    }]
+  })
+  tags = {
+    Project = "aipcc-akka"
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "ecs_task_execution_attach" {
+  role       = aws_iam_role.ecs_task_execution.name
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
+}
+
+resource "aws_security_group" "akka_sg" {
+  name        = "akka-internal-sg"
+  description = "Allow internal access to ECS task on port 5000"
+  vpc_id      = "vpc-0ff68e46a55526721"
```

Comment:
Does it make more sense to pull the VPC value from a non-tracked .tfvars file instead of hardcoding it?

Reply:
Yes, added an example tfvars file.